tests/resources/Engine.py

```diff
@@ -517,14 +520,15 @@ def create_template_file(poller: int, typ: str, what: str, ids):
             content = ""
             idx = 1
             for i in ids:
-                content += """define {} {{
-name                   {}_template_{}
-{}               {}
+                content += f"""define {typ} {{
```

Comment:
A template should be always defined without alias. It is a bad practice to define it in a template.
So No need to modify this function here.